src/uu/stat/src/stat.rs

```diff
@@ -471,24 +473,10 @@ impl Stater {
     }
 
     fn new(matches: &ArgMatches) -> UResult<Self> {
-        let mut files: Vec<String> = matches
+        let files = matches
             .values_of(ARG_FILES)
```

Comment:
If we're using `OsString` anyway, this can be `.values_of_os(ARG_FILES)`, right?

Reply:
Thanks for making me aware of this. I changed it.